person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_reader @id
+  attr_accessor @name, @age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    is_of_age? || @parent_permission
```

Comment:
- [ ] Add parentheses here, in order to avoid causing syntactic ambiguities and confusing corner cases.
`is_of_age?()`

Reply:
@GzimAsani Kindly note that if I use parenthesis, linters flags an error that's why I didn't add it.